components/cards/CardProject.tsx

```diff
@@ -0,0 +1,93 @@
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent'
+import Typography from '@mui/material/Typography'
+
+import Button from '@mui/material/Button'
+import Box from '@mui/material/Box'
+import Stack from '@mui/material/Stack'
+import StateButton from './StateButton'
+import Placeholder from '../../assets/placeholder-img.png'
+
+type CardProjectProps = {
+  title: string
+  partner: string
+  programAreas: string[]
+  description: string
+  status: 'active' | 'recruiting' | 'complete'
+  projectLink: string
+  duration: { start: string; end: string } // unsure how to handle dates at this point
+}
+
+const CardProject = ({
+  title,
+  partner,
+  programAreas,
+  description,
+  status,
+  projectLink,
+  duration,
+}: CardProjectProps) => {
+  return (
```

Comment:
tip. you don't need to have return, just:

```
const CardProject = ({...}) => (<Card> ... </Card>)

```
If there's no react logic in there.

Reply:
ah thanks for catching that!